modules/telemetry/telemetrymodule.cpp

```diff
@@ -0,0 +1,314 @@
+ * Copyright (c) 2014-2024                                                               *
+ *                                                                                       *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this  *
+ * software and associated documentation files (the "Software"), to deal in the Software *
+ * without restriction, including without limitation the rights to use, copy, modify,    *
+ * merge, publish, distribute, sublicense, and/or sell copies of the Software, and to    *
+ * permit persons to whom the Software is furnished to do so, subject to the following   *
+ * conditions:                                                                           *
+ *                                                                                       *
+ * The above copyright notice and this permission notice shall be included in all copies *
+ * or substantial portions of the Software.                                              *
+ *                                                                                       *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,   *
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A         *
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT    *
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF  *
+ * CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE  *
+ * OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                                         *
+ ****************************************************************************************/
+
+#include <modules/telemetry/telemetrymodule.h>
+
+#include <modules/telemetry/include/general/anglemodetelemetry.h>
+#include <modules/telemetry/include/general/cameratelemetry.h>
+#include <modules/telemetry/include/general/focustelemetry.h>
+#include <modules/telemetry/include/general/nodestelemetry.h>
+#include <modules/telemetry/include/general/timetelemetry.h>
+#include <modules/telemetry/include/specific/planetscomparesonification.h>
+#include <modules/telemetry/include/specific/planetsoverviewsonification.h>
+#include <modules/telemetry/include/specific/planetssonification.h>
+#include <ghoul/logging/logmanager.h>
+#include <openspace/camera/camera.h>
+#include <openspace/documentation/documentation.h>
+#include <openspace/engine/globals.h>
+#include <openspace/engine/globalscallbacks.h>
+#include <openspace/engine/windowdelegate.h>
+#include <openspace/rendering/renderengine.h>
+#include <openspace/scene/scene.h>
+
+namespace {
+    constexpr std::string_view _loggerCat = "TelemetryModule";
+
+    // The default OSC reciver is SuperCollider with these default values.
+    // However, the user can define any reciver in the openspace.cfg file as the
+    // ModuleConfiguration for the Telemetry module.
```

Comment:
```suggestion
    // The default OSC receiver is SuperCollider with these default values.
    // However, the user can define any receiver in the openspace.cfg file as the
    // ModuleConfiguration for the Telemetry module.
```

Then we should not mention "over the OSC connection" in the comments in all the other files.